iothub/device/src/DeviceClient.ConventionBasedOperations.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
wondering reasoning behind the naming DeviceClient.xxx

Reply:
Since we already have a class called DeviceClient.cs, the other partial class needed to have a different name. The "convention-based-operation" part of the name is to signify that this file holds the APIs that follow a particular convention for their payload.